### PR TITLE
Fixed #150: Added two new speakers

### LINKED ISSUE #150: Add Two New Featured Speakers on the very top of the section
Please add the follow two speakers on the top of the featured speakers section. These two should be in position 1 and 2. 

1. Chris Aniszczyk
Cloud Native Computing Foundation
Chris is currently a VP at the Linux Foundation focused on developer relations and running the Open Container Initiative (OCI) / Cloud Native Computing Foundation (CNCF). Furthermore, he's a partner at Capital Factory where he focuses on mentoring, advising and investing in open source and infrastructure focused startups
chrisaniszczyk

2. Michael Christen
SUSI.AI 
Michael is the founder of SUSI AI, loklak and, the creator of the Peer-to-Peer Search engine YaCy. He is a Big Data Engineer consulting for some of the largest corporate players in Germany on search technology and digital transformation strategies. He is also architect of large search portals like the German Digital Library.
michaelchristen

Where? 
screenshot from 2018-01-10 11-56-13

### PR DESCRIPTION
**Pull request by samyakbvs**
I have added two new speakers on the very top as @hpdang requested

### CODE REVIEW

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/samyak-2018.fossasia.org/

**Comment by hpdang** [comment]
@samyakbvs looks like there is an alignment error. See below

screenshot from 2018-01-11 15-50-34

**Comment by samyakbvs** [comment]
IK @hpdang but I tried and I am not able to solve it

**Comment by simsausaurabh** [comment]
And you have only added twitter link on hover image of both the speakers.

File: index.html [hunk 1]
```diff
@@ -1591,7 +1591,49 @@ <h1>Featured Speakers</h1>
                         </div>
                     </div>
 
-                    <div class="row">
+                    <div class="row>
```
Comment (simsausaurabh) [inline index.html]: <div class="row"> you have missed double quotes

Comment (simsausaurabh) [inline index.html]: <div class="row"> you have missed double quotes.

**Comment by simsausaurabh** [comment]
Add these under social links tag for Chris: 

`<a target="default" href="http://aniszczyk.org/"><i class="icon fa fa-external-link"></i></a>
                                                <a target="default" href="https://medium.com/@cra_82903"><i class="icon fa fa-medium"></i></a>
                                                <a target="default" href="https://twitter.com/cra"><i class="icon social_twitter"></i></a>
                                                <a target="default" href="https://www.linkedin.com/in/caniszczyk/"><i class="icon social_linkedin"></i></a>
                                                <a target="default" href="https://github.com/caniszczyk"><i class="icon fa fa-github"></i></a>`

and under social links for tag of Michael add: 

`<a target="default" href="https://yacy.net/en/index.html"><i class="icon fa fa-external-link"></i></a>
                                                <a target="default" href="https://twitter.com/0rb1t3r"><i class="icon social_twitter"></i></a>
                                                <a target="default" href="https://www.linkedin.com/in/orbiter/"><i class="icon social_linkedin"></i></a>
                                                <a target="default" href="https://github.com/orbiter/"><i class="icon fa fa-github"></i></a>`

**Comment by hpdang** [comment]
@simsausaurabh can you make a new PR? We need this to be done today

**Comment by simsausaurabh** [comment]
@hpdang yes sure, I'm working on it.

**Comment by simsausaurabh** [comment]
@hpdang I have made a new PR for #152 
Please close this PR.

**Pull request closed without merging**